### PR TITLE
Fewer media queries in CommunityAbout section

### PR DESCRIPTION
**Pull request by zoeabryant**
to cover @AndrewBestbier 's comment on PR #231

### CODE REVIEW

**Comment by lpil** [comment]
Looks good.

File: shared/components/CommunityAbout/index.scss [hunk 1]
```diff
@@ -21,16 +15,19 @@
       margin-bottom: 0px;
     }
 
-    @media (min-width: 768px) {
+    @media (min-width: $tablet-portrait-width) {
```
Comment (AndrewBestbier) [inline shared/components/CommunityAbout/index.scss]: Looks good :) Were there other examples of this in the other css files?

Comment (zoeabryant) [inline shared/components/CommunityAbout/index.scss]: Good point!

.EventDetails, .NextCommunityEvent, .NextConferenceEvent and .TicketStatus refers to 768px

I will go throw and replace these.


.SiteFooter has 750px (but @lpil did tell me there was a particular reason for that)

Comment (zoeabryant) [inline shared/components/CommunityAbout/index.scss]: confirmed over slack - andrew meant examples of media queries being in confusing places.

I'm still gonna get rid of 768px media queries!

**Comment by zoeabryant** [comment]
This has the +1 of @JunBadger on behalf of the design team

**Pull request merged**